pom.xml

```diff
@@ -266,7 +265,7 @@
         <gson.version>2.10</gson.version>
         <jnr.version>3.1.15</jnr.version>
         <protobuf.version>3.21.9</protobuf.version>
-        <osgi.version>8.0.0</osgi.version>
+        <osgi.version>6.0.0</osgi.version>
```

Comment:
I do not see this being used- I suggest removing it

Reply:
it is referenced at line 3338